lib/pf/UnifiedApi/Controller/Auditlogs.pm

```diff
@@ -0,0 +1,52 @@
+package pf::UnifiedApi::Controller::Auditlogs;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Auditlogs -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Auditlogs
+
+=cut
+
+use strict;
+use warnings;
+use Mojo::Base 'pf::UnifiedApi::Controller::Crud';
+use pf::dal::radius_audit_log;
+
+has dal => 'pf::dal::radius_audit_log';
+has url_param_name => 'auditlog_id';
```

Comment:
I'd go with radius_audit_log_id so that it stays consistent with the table name

Reply:
The automagic of url parameter always matches the singular name of the controller.
So if you want to change it then change name of the controller